spec/game_spec.rb

```diff
@@ -0,0 +1,3 @@
+describe Game do
+
+end
```

Comment:
I cant claim credit for any of the stuff in my code but I think the testing approach was smart and may have been useful for yourself too.  Eg 'it scores all 0's', or 'it scores all 1's', 'it scores a strike'
Maybe asking ourselves these questions in our tests can actually help us with what code we should write.